usr/bin/whonix-workstation-firewall

```diff
@@ -39,18 +39,21 @@ variables_defaults() {
     ## Would fail if netvm is set to 'none',
     ## which is the case in Qubes R4 TemplateVMs.
     [ -n "${GATEWAY_IP:-}" ] || GATEWAY_IP="$(qubesdb-read /qubes-gateway 2> /dev/null)" || GATEWAY_IP="127.0.0.1"
+    [ -n "${GATEWAY_IP6:-}" ] || GATEWAY_IP6="::ffff:$(qubesdb-read /qubes-gateway6 2> /dev/null)" || GATEWAY_IP6="::1"
```

Comment:
Maybe I misunderstood something, but I was under the impression that the `/qubes-gateway6` qubesdb entry contained an IPv6 address. This looks like it's being used as an IPv4 address though.

Reply:
Based on https://github.com/QubesOS/qubes-doc/blob/d9a6e20f3a1ad3ca5215059267df099bfecc6728/user/advanced-topics/standalones-and-hvms.rst?plain=1#L258 it does look like `/qubes-gateway6` is a real IPv6 address. Adjusting accordingly.